helios-services/pom.xml

```diff
@@ -554,6 +554,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
```

Comment:
How come this is needed? The same config is in `<pluginManagement>` of the parent pom